src/containers/WebComponentLoader.test.js

```diff
@@ -86,7 +86,7 @@ describe("When no user is in state", () => {
         project: {
           components: [],
         },
-        hasShownSavePrompt: false,
+        hasShownSavePrompt: true,
```

Comment:
I'm not sure why this has been changed 🤔  `hasShownSavePrompt` should stay as it was and with `showSavePrompt` defaulting to false these tests should work without change unless I'm missing something

Reply:
I guess it's because we're now defaulting to not showing the save prompt, so the change on line 109 of `WebComponentLoader` means that we're passing this in as `true` to stop the prompt from showing unless they've specified that they want it... Maybe the naming is not the best though?